packages/forms-web-app/src/views/selected-appeal/appeal.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "layouts/no-banner-link/main.njk" %}
+
+ {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+ {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+ {% set title="
+ Appeal " + appeal.appealNumber + " - Appeal a planning decision - GOV.UK" %}
+
+ {# is this variable necessary? #}
+ {% set headlineData = appeal.headlineData %}
+
+ {% block pageTitle %}
+   {{ title }}
+ {% endblock %}
+
+ {% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+       {% if errorSummary %}
+         {{ govukErrorSummary({
+           titleText: "There is a problem",
+           errorList: errorSummary
+         }) }}
+       {% endif %}
+
+       <h1 class="govuk-heading-l">Appeal {{appeal.appealNumber}}</h1>
+
+			<style>
+				.appeal-details .govuk-summary-list__key,
+				.appeal-details .govuk-summary-list__value,
+				.appeal-details .govuk-summary-list__actions {
+					padding-top: .2rem;
+					padding-bottom: 0;
+				}
+				{# I can't tell what this is doing #}
+				.app-task-list__section {
+					padding-right: 140px
+				}
+			</style>
```

Comment:
Do we need to deviate from default styles? I assume this is to match something on the prototype?

if we do need something, the rest of the css is in sass files.